fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/RelationTransformer.java

```diff
@@ -430,7 +430,9 @@ public LogicalPlan visitTable(TableRelation node, ExpressionMapping context) {
             scanOperator = new LogicalHiveScanOperator(node.getTable(), node.getTable().getType(),
                     colRefToColumnMetaMapBuilder.build(), columnMetaToColRefMap, Operator.DEFAULT_LIMIT, null);
         } else if (Table.TableType.ICEBERG.equals(node.getTable().getType())) {
-            ((IcebergTable) node.getTable()).refreshTable();
+            if (!((IcebergTable) node.getTable()).isCatalogTbl()) {
```

Comment:
why only the non-catalog table should perform `refresh` ops

Reply:
catalog table will load each time when get table during query analyze phase